content/docs/command-reference/fetch.md

```diff
@@ -42,20 +52,10 @@ project's cache                  ++ | dvc pull |
  workspace
 ```
 
-Fetching could be useful when first checking out a <abbr>DVC project</abbr>,
-since files tracked by DVC should already exist in remote storage, but won't be
-in the project's <abbr>cache</abbr>. (Refer to `dvc remote` for more information
-on DVC remotes.) These necessary data or model files are listed as
-<abbr>dependencies</abbr> or <abbr>outputs</abbr> in a target
-[stage](/doc/command-reference/run) (in `dvc.yaml`) or `.dvc` file, so they are
-required to [reproduce](/doc/tutorials/get-started/data-pipelines#reproduce) the
-corresponding [pipeline](/doc/command-reference/pipeline).
-
-`dvc fetch` ensures that the files needed for a stage or `.dvc` file to be
-[reproduced](/doc/tutorials/get-started/data-pipelines#reproduce) exist in
-cache. If no `targets` are specified, the set of data files to fetch is
-determined by analyzing all `dvc.yaml` and `.dvc` files in the current branch,
-unless `--all-branches` or `--all-tags` is specified.
+Fetching is useful when first checking out a <abbr>DVC project</abbr> for
```

Comment:
not very clear. Previously we had some reference to the reproduce. There are few cases when fetch is needed:

* across different branches to compare functions to work - `* diff` commands that depend on DVC tracked files
* handy if you want for some reason to keep workspace clean (e.g. it's dirty already and you don't want to merge/commit yet
* when linking is not needed for some reason? e.g. shared cache?